src/figuresClasses.ts

```diff
@@ -1,19 +1,91 @@
-export interface Figure {
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blue',
+}
 
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
```

Comment:
According to the checklist:  ![1x](https://github.com/mate-academy/ts_figures_classes/assets/146858277/64a745e2-0207-4f74-a177-6f876e0d6b54)

2. [CODE STYLE] - Enum constant names usually start with capital letter. Values can have any case that's necessary.

BAD EXAMPLE:
```
export enum Directions {
  up = 'UP',
  down = 'DOWN',
  left = 'LEFT',
  right = 'RIGHT',
}
```

GOOD EXAMPLE:
```
export enum Directions {
  Up = 'UP',
  Down = 'DOWN',
  Left = 'LEFT',
  Right = 'RIGHT',
}
```

Reply:
not fixed 😥